staging/src/k8s.io/kubectl/pkg/cmd/debug/debug.go

```diff
@@ -107,6 +111,7 @@ type DebugOptions struct {
 	Attach          bool
 	Container       string
 	CopyTo          string
+	Capabilities    string
```

Comment:
Store capabilities here in options. You can do the conversion in `Complete()`.

Reply:
Thanks, completed